build/mac/find_sdk.py

```diff
@@ -10,15 +10,25 @@
   python find_sdk.py 10.6  # Ignores SDKs < 10.6
 """
 
+import errno
 import os
 import re
 import subprocess
 import sys
 
-
 from optparse import OptionParser
 
 
+def mkdir_p(path):
```

Comment:
Is there any way to avoid copying this to several places?

Reply:
Yeah. I also realize that I need to fix this so it replaces the symlink incase you e.g. update or move Xcode for some reason.